modules/trusted-firmware-m/Kconfig

```diff
@@ -78,6 +78,7 @@ config PM_PARTITION_SIZE_TFM
 	hex
 	prompt  "Memory reserved for TFM" if !TFM_MINIMAL
 	default 0x10000 if TFM_MINIMAL
+	default 0x60000 if TFM_REGRESSION_S
```

Comment:
It is better practice for users to know of features than for features to know of users.

I see we have a prompt, so could the regression test do

CONFIG_TFM_REGRESSION_S=0x60000
?

Reply:
That would have to be a noup to sdk-zephyr then.

Reply:
This is a temporary hack. Will be fixed soon.